db_helper.php

```diff
@@ -1,98 +1,129 @@
 <?php
+if (!file_exists('config.php')) {
+    exit('Please copy \'config.sample.php\' to \'config.php\'');
+}
+require_once 'config.php';
 
+if (Config::DEBUG) {
     mysqli_report(MYSQLI_REPORT_ERROR | MYSQLI_REPORT_STRICT);
+} else {
+    mysqli_report(MYSQLI_REPORT_OFF);
+}
 
-    class DBHelper {
+class DBHelper
+{
+    private $_connection;
```

Comment:
again, out of curiosity: why the `_` prefix when it is already declared as private. I know that in Python, you start with `_` to softly mark something as private because `private` does not exist there.

Reply:
This change was according to the output of [`phpcs`](https://github.com/squizlabs/PHP_CodeSniffer). Adding `_` is just a visual reminder that it's a private variable. Not defined in PSR.